packages/headless/src/api/commerce/product-listings/product-listing-params.ts

```diff
@@ -26,16 +27,16 @@ export type ProductListingRequestParam = {
     requests: AnyFacetRequest[];
     options?: FacetOptions;
   };
-  sort?: {
-    by: 'RELEVANCE' | 'FIELDS';
-    fields?: [
-      {
-        direction: 'ASC' | 'DESC';
-        name: string;
-      }
-    ];
-  };
-};
+  sort?: ProductListingSortRequestParam;
+}
+
+export interface ProductListingSortRequestParam {
+  by: 'relevance' | 'fields';
+  fields?: Array<{
+    direction: 'ASC' | 'DESC';
```

Comment:
these should be lowercased
![image](https://user-images.githubusercontent.com/33127602/132704736-0816ebc1-f72b-48aa-a7bf-6a9992dbcabf.png)